src/mapped-types.ts

```diff
@@ -576,3 +576,41 @@ export type Optional<T extends object, K extends keyof T = keyof T> = Omit<
   K
 > &
   Partial<Pick<T, K>>;
+
+/**
+ * $ValuesType
```

Comment:
Should be ValuesType, without $

Reply:
Oops I forgot to change the examples when I copied it from the `$ValuesType` I wrote at first.

I've removed that `$`.